.github/workflows/build-and-publish-release.yml

```diff
@@ -0,0 +1,102 @@
+          echo "IMAGE_TAG_UI=ghcr.io/opencost/opencost-ui:${{ steps.sha.outputs.OC_SHORTHASH }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG_UI_LATEST=ghcr.io/opencost/opencost-ui:latest" >> $GITHUB_ENV
+          echo "IMAGE_TAG_UI_VERSION=ghcr.io/opencost/opencost-ui:${{ inputs.release_version }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG_QUAY=quay.io/kubecost1/kubecost-cost-model:${{ steps.sha.outputs.OC_SHORTHASH }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG_LATEST_QUAY=quay.io/kubecost1/kubecost-cost-model:latest" >> $GITHUB_ENV
+          echo "IMAGE_TAG_VERSION_QUAY=quay.io/kubecost1/kubecost-cost-model:prod-v${{ inputs.release_version }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG_UI_QUAY=quay.io/kubecost1/opencost-ui:${{ steps.sha.outputs.OC_SHORTHASH }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG_UI_LATEST_QUAY=quay.io/kubecost1/opencost-ui:latest" >> $GITHUB_ENV
+          echo "IMAGE_TAG_UI_VERSION_QUAY=quay.io/kubecost1/opencost-ui:${{ inputs.release_version }}" >> $GITHUB_ENV
+     
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      
+      - name: Set up just
+        uses: extractions/setup-just@v1
+
+      ## Install manifest-tool, which is required to combine multi-arch images
+      ## https://github.com/estesp/manifest-tool
+      - name: Install manifest-tool
+        run: |
+          mkdir -p manifest-tool
+          pushd manifest-tool
+          wget -q https://github.com/estesp/manifest-tool/releases/download/v2.0.8/binaries-manifest-tool-2.0.8.tar.gz
+          tar -xzf binaries-manifest-tool-2.0.8.tar.gz
+          cp manifest-tool-linux-amd64 manifest-tool
+          echo "$(pwd)" >> $GITHUB_PATH
+
+      - name: Login to Quay
+        uses: docker/login-action@v3
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      
+      - name: Build and push (multiarch) OpenCost
+        working-directory: ./opencost
+        run: |
+          just build '${steps.tags.outputs.IMAGE_TAG}'
+          just build '${steps.tags.outputs.IMAGE_TAG_LATEST}'
+          just build '${steps.tags.outputs.IMAGE_TAG_VERSION}'
+          just build '${steps.tags.outputs.IMAGE_TAG_QUAY}'
+          just build '${steps.tags.outputs.IMAGE_TAG_LATEST_QUAY}'
+          just build '${steps.tags.outputs.IMAGE_TAG_VERSION_QUAY}'
+      - name: Build and push (multiarch) OpenCost UI
+        working-directory: ./opencost/ui
+        run: |
+          just build '${steps.tags.outputs.IMAGE_TAG_UI}'
+          just build '${steps.tags.outputs.IMAGE_TAG_UI_LATEST}'
+          just build '${steps.tags.outputs.IMAGE_TAG_UI_VERSION}'
+          just build '${steps.tags.outputs.IMAGE_TAG_UI_QUAY}'
+          just build '${steps.tags.outputs.IMAGE_TAG_UI_LATEST_QUAY}'
+          just build '${steps.tags.outputs.IMAGE_TAG_UI_VERSION_QUAY}'
```

Comment:
`crane` is a great option here to avoid extra docker build work
Install in a separate step with
```yaml
      - name: Install crane
        uses: imjasonh/setup-crane@v0.1
```

and then you can do something like this

```sh
just build '${steps.tags.outputs.IMAGE_TAG}'
crane copy  '${steps.tags.outputs.IMAGE_TAG}' '${steps.tags.outputs.IMAGE_TAG_LATEST}'
crane copy  '${steps.tags.outputs.IMAGE_TAG}' '${steps.tags.outputs.IMAGE_TAG_VERSION}'
# ...etc
```

Reply:
added